### ISSUE TITLE
5.0.12 Failed to load image

### ISSUE THREAD

**Issue by Kenobi77** [grokability/snipe-it#9039]
#### Please confirm you have done the following before posting your bug report:

- [x] I have enabled debug mode 
- [x] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**

Images upload fine and show in public\uploads\models but when I click on then in asses is says image filed to load 
**To Reproduce**
1  Click on Asset\Models then Edit and you can see the uploaded image.
2 unable to see the image anywhere else though.  
3 Select 'List all assets' and get failed ot load image from uploads\asset\asset-image-jkhkhfa.jpg

**Expected behavior**
Should display image

**Screenshots**

**Server (please complete the following information):**
 - Snipe-IT Version 5.0.12
 - OS: [e.g. Ubuntu, CentOS] Server 2016
 - Web Server: [e.g. Apache, IIS] IIS
 - PHP Version 7.4.14

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Mac Os
 - Browser [e.g. chrome, safari] safari (Happens on all browsers though)
 - Version [e.g. 22] 13.1.2

**Smartphone (please complete the following information):**

**Error Messages**
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
- If a stacktrace is provided in the error, include that too.
- Any errors that appear in your browser's error console.
- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo.
- Include any additional information you can find in `storage/logs` and your webserver's logs.
- Include the output from `php -m` (this should display what modules you have enabled.)

**Additional context**
- Is this a fresh install or an upgrade?  Fresh Install
- What OS and web server you're running Snipe-IT on Server 2016 IIS 10
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc) Manual install following guide in snipe-it site
- Include what you've done so far in the installation, and if you got any error messages along the way. All good with the install.
- Indicate whether or not you've manually edited any data directly in the database No.

Add any other context about the problem here.  Made sure that public, storage, cache and Windows Temp has IUSR full access (also added everyon for the time being but that hasn't worked either)

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

**Comment by Kenobi77** [comment]
nothing in storage/logs

**Comment by djshoxxx** [comment]
I am having likely the same issue, posted here: https://github.com/snipe/snipe-it/issues/9090

kenobi77, if you inspect the broken image, where does it point too?

**Comment by Kenobi77** [comment]
Asset-image

**Comment by djshoxxx** [comment]
> Asset-image

yeah but can you post the exact URL that it points too? for example, one of my broken images points to: http://192.168.1.2:8087/hardware/1/qr_code

**Comment by Kenobi77** [comment]
Ah sorry please see below
Http://192.168.0.53/uploads/assets/asset-image-nJMLblahblah.jpg

I made the blah blah part up but hopefully you get the gist :-)

**Comment by djshoxxx** [comment]
if you add in the port number to the URL does your image start working? for example mine is:
http://192.168.1.2:8087/hardware/1/qr_code
and its broken

but when I change it to the correct port
http://192.168.1.2:8086/hardware/1/qr_code

it starts working. (I thought this was a typo I made in the env file at first, but couldnt see anywhere where I had typed 8087... anyways..)

if you go 
Http://192.168.0.53:XX/uploads/assets/asset-image-nJMLblahblah.jpg

where XX is the port number that your snipeIT uses, does it start working? Im not a dev for this project, but am curious as to the problems, yours and mine both
